### ISSUE TITLE
Panic when missing argument to string flag

### ISSUE THREAD

**Issue by mpizenberg** [ksk001100/seahorse#48]
I was just playing with the API and it appears that not providing the command line argument to a string flag triggers a panic with message:
```
thread 'main' panicked at 'assertion failed: index < len', <::core::macros::panic macros>:2:2
```
Cool logo by the way!

**Comment by rnitta** [comment]
@mpizenberg 
Thanks for reporting!

I found that code here is the cause.

https://github.com/ksk001100/seahorse/blob/d63e9bfa18e72385bd6aabe98df572994cc8f49d/src/context.rs#L36

This error reproduces when you  pass no value to a flag (unless it is bool typed)....
🤔🤔🤔 @ksk001100  

---

> Cool logo by the way!

😊😊😊😊😊😊😊😊😊😊

**Comment by ksk001100** [comment]
@mpizenberg 
Thank you for reporting the issue.
This bug will be fixed soon.

**Comment by rnitta** [comment]
Resolved by #49
